src/components/ContentHighlights/HighlightStepper/ContentHighlightStepper.jsx

```diff
@@ -46,6 +53,7 @@ const ContentHighlightStepper = ({ enterpriseId }) => {
   const { location } = history;
   const [currentStep, setCurrentStep] = useState(steps[0]);
   const [isPublishing, setIsPublishing] = useState(false);
+  const [isOpen, open, close] = useToggle(false);
```

Comment:
nit: `[isCloseAlertOpen, openCloseAlert, closeCloseAlert]` or similar to be a tad more explicit